desktop/mode.py

```diff
@@ -29,7 +29,8 @@ def __init__(self, kind, **info):
         self.kind = kind
```

Comment:
I am not a fan of this variable name bc I cannot tell what it is for.

Reply:
it's a legacy variable name with possible values "SELECTED", "MULTIPLE", "QUESTION", "PROMPT", and "DRAW". I'm open to alternative names although I'll note that this variable is everywhere in both desktop and browser

Reply:
Maybe it's not worth changing in this PR but it definitely seems like it could be renamed for clarity. We can chew on it and make another PR in the future.

Reply:
I think that could be a good opportunity to also turn the fixed value strings into class attributes.